tests/integration/test_fim/test_synchronization/test_sync_disabled_win32.py

```diff
@@ -95,16 +95,21 @@
 
 
 # fixtures
-
 @pytest.fixture(scope='module', params=configurations)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
-# Tests
+@pytest.fixture(scope='module')
+def create_a_file(get_configuration):
+    """Create a file previous to restart syscheckd"""
+    create_file(REGULAR, test_directories[0], 'damaris.txt')
```

Comment:
Please, change the text name (damaris.txt) for something representative.

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/2802/commits/4c3da959cfee883a9065f54a44ae41c3fb74659e